src/platform/ESP32/ESP32ThreadBorderRouterDelegate.h

```diff
@@ -0,0 +1,94 @@
+/*
+ *
+ *    Copyright (c) 2024 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "openthread/dataset.h"
+#include <app-common/zap-generated/cluster-enums.h>
+#include <app/clusters/thread-border-router-management-server/thread-br-delegate.h>
+#include <inet/IPAddress.h>
+#include <lib/core/CHIPError.h>
+#include <lib/support/Span.h>
+#include <openthread/netdiag.h>
+
+using chip::MutableByteSpan;
+using chip::MutableCharSpan;
+using chip::Span;
```

Comment:
Please don't put `using` in headers at toplevel.

Especially because all the stuff below is inside `namespace chip` anyway, so the barewords should work.

Reply:
Removed